.github/workflows/cfn-nag.yml

```diff
@@ -23,44 +23,55 @@ jobs:
         uses: ./.github/actions/baseAction
 
       - name: cdk synth swb-reference
+        env:
```

Comment:
https://github.com/aws-solutions/solution-spark-on-aws/actions/runs/3625598813/jobs/6113813300. There are 45 voilations detected. Please check the `echo voilation count` step

Reply:
When do we plan to enable?

Reply:
I am waiting for SWB to give me a green light. I was informed that they have a backlog item to work on this. If I enable it now, it will block all the PRs